### ISSUE TITLE
padding-line-between-statements crash on code "};"

### ISSUE THREAD

**Issue by andoras** [eslint/eslint#9051]
Environment:
* ESLint 4.3
* Parser espree
* Latest NPM LTS

I used the rule `padding-line-between-statements` in my configuration as follows (of course for reproduction you should remove the comment before the first parameter).

**Configuration**
```
	"padding-line-between-statements": [
        "warn",
        // This bringst ESlint to crash if the code contains "};" -->   { "blankLine": "always", "prev": "block-like", "next": "*" },
        { "blankLine": "always", "prev": "*", "next": "class" },
        { "blankLine": "always", "prev": "directive", "next": "*" },
        { "blankLine": "always", "prev": "export", "next": "*" },
        { "blankLine": "always", "prev": "import", "next": "*" }
```

**What did you expect to happen?**
ESlint should not crash if a code block ends with a "};" where the ";" is unneccessary.

**What actually happened? Please include the actual, raw output from ESLint.**
Executing ESLint on a .js file containing a code block with a ";" behind the "}" produces this output:
```
TypeError: Cannot read property 'value' of null
    at Object.isClosingBraceToken (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\ast-utils.js:364:17)
    at isBlockLikeStatement (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\rules\padding-line-between-statements.js:96:36)
    at Object.test (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\rules\padding-line-between-statements.js:331:37)
    at match (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\rules\padding-line-between-statements.js:473:41)
    at getPaddingType (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\rules\padding-line-between-statements.js:488:21)
    at Linter.verify (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\rules\padding-line-between-statements.js:551:30)
    at emitOne (events.js:96:13)
    at Linter.emit (events.js:188:7)
    at NodeEventGenerator.applySelector (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\util\node-event-generator.js:265:26)
    at NodeEventGenerator.applySelectors (D:\TEMP\_jslinterpack\linter\node_modules\eslint\lib\util\node-event-generator.js:292:22)
```

**Comment by jukebox017** [comment]
Possible duplicate of #8839 which has been closed by #8748

**Comment by not-an-aardvark** [comment]
Closing as duplicate of #8839
